packages/plugin-module-federation/src/cli/index.ts

```diff
@@ -0,0 +1,82 @@
+import { parseOptions } from '@module-federation/enhanced';
+import { ModuleFederationPlugin } from '@module-federation/enhanced/rspack';
+
+import { isRegExp, isRequiredVersion } from '../utils/index';
+
+import type {
+  moduleFederationPlugin,
+  sharePlugin,
+} from '@module-federation/sdk';
+import type { RsbuildPlugin } from '@rsbuild/core';
+
+type ModuleFederationOptions =
+  moduleFederationPlugin.ModuleFederationPluginOptions;
+
+const PLUGIN_MODULE_FEDERATION_NAME = 'rsbuild:module-federation';
+const pkgName = '@rsbuild/plugin-module-federation';
```

Comment:
If the plugin name is `@rsbuild/plugin-module-federation`, it should work with Rsbuild-based web apps too.